specs/sharding/beacon-chain.md

```diff
@@ -580,226 +496,144 @@ def process_operations(state: BeaconState, body: BeaconBlockBody) -> None:
     for_ops(body.attestations, process_attestation)
     for_ops(body.deposits, process_deposit)
     for_ops(body.voluntary_exits, process_voluntary_exit)
+```
+
+#### Intermediate Block Operations
+
+```python
+def process_intermediate_block_attestations(state: BeaconState, body: IntermediateBlockBody) -> None:
 
-    # TODO: to avoid parallel shards racing, and avoid inclusion-order problems,
-    #  update the fee price per slot, instead of per header.
-    # state.shard_sample_price = compute_updated_sample_price(state.shard_sample_price, ?, shard_count)
+    for attestation in block.body.attestations:
+    process_attestation(state, block.body.attestation)
 ```
 
-##### Extended Attestation processing
+#### Intermediate Block Bid Commitment
 
 ```python
-def process_attestation(state: BeaconState, attestation: Attestation) -> None:
-    altair.process_attestation(state, attestation)
-    process_attested_shard_work(state, attestation)
+def process_intermediate_block_bid_commitment(state: BeaconState, body: IntermediateBlockBody) -> None:
```

Comment:
Is any state modifications supposed be added here later? If not then may be the better function name would be `verify_intermediate_block_bid_commitment`

Reply:
Yeah good idea